src/osx/dataview_osx.cpp

```diff
@@ -233,76 +233,65 @@ void wxOSXDataViewModelNotifier::Resort()
 
 void wxOSXDataViewModelNotifier::AdjustRowHeight(wxDataViewItem const& item)
 {
-  if ((m_DataViewCtrlPtr->GetWindowStyle() & wxDV_VARIABLE_LINE_HEIGHT) != 0)
+  if (!m_DataViewCtrlPtr->IsFrozen() ||  ((m_DataViewCtrlPtr->GetWindowStyle() & wxDV_VARIABLE_LINE_HEIGHT) != 0))
   {
-      wxDataViewModel *model = GetOwner();
+    wxDataViewModel *model = GetOwner();
+
+    int height = ((m_DataViewCtrlPtr->GetWindowStyle() & wxDV_VARIABLE_LINE_HEIGHT) != 0)
+        ? 20
+        : m_DataViewCtrlPtr->GetDataViewPeer()->GetRowHeight();
+    int origHeight = height;
```

Comment:
Sorry, I don't understand what is this for, could you please explain? Where/how does `origHeight` become different from `height`? It looks like neither of them ever changes after the initial assignment, there used to be an assignment to `height` before but it has disappeared now...

Reply:
A little lower, there's `height = wxMax( height, renderer->GetSize().y );` within the loop iterating over columns. Later, it is used to avoid explicit call to `GetDataViewPeer()->SetRowHeight()` if the height didn't change.